pkgs/development/python-modules/pyserial-asyncio/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k
+, pyserial }:
+
+buildPythonPackage rec {
+  pname = "pyserial-asyncio";
+  version = "0.4";
+
+  disabled = !isPy3k; # Doesn't support python older than 3.4
```

Comment:
python2 is EOL in 3 weeks, do they areally no support python3?

looks like they haven't had activity on the repo for over 18months....

Reply:
@jonringer That's not what this line implies though? The code say that it's disabled if it's not python 3, i.e. python2 doesn't work at all. Also they state somewhere (I don't remember where I read it because it was 8 days ago that I created the PR) that they require python 3.4 or newer (which the comment states).

Reply:
i read this too quickly, my bad